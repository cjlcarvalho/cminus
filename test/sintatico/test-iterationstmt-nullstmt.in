void main(void){
	while(a==b+1);
	return;
}
