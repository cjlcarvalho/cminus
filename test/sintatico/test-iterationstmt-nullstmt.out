[program 
  [decfunc 
    [void]
    [main]
    [paramlist]
    [block 
      [while 
        [== [var [a]] 
          [+ [var [b]] [1]]]
        [null]
      ]
      [return ]
    ]
  ]
]
